src/Platforms/OraclePlatform.php

```diff
@@ -603,16 +598,16 @@ public function getAlterTableSQL(TableDiff $diff): array
             if ($countChangedProperties > ($columnHasChangedComment ? 1 : 0)) {
                 $newColumnProperties = $newColumn->toArray();
 
-                $oldSQL = $this->getColumnDeclarationSQL('', $oldColumn->toArray());
-                $newSQL = $this->getColumnDeclarationSQL('', $newColumnProperties);
+                $oldSQL = $this->getColumnDeclarationSQL($oldColumn->toArray());
+                $newSQL = $this->getColumnDeclarationSQL($newColumnProperties);
 
                 if ($newSQL !== $oldSQL) {
                     if (! $columnDiff->hasNotNullChanged()) {
                         unset($newColumnProperties['notnull']);
-                        $newSQL = $this->getColumnDeclarationSQL('', $newColumnProperties);
+                        $newSQL = $this->getColumnDeclarationSQL($newColumnProperties);
                     }
 
-                    $modifyColumnSQL[] = $newColumn->getObjectName()->toSQL($this) . $newSQL;
+                    $modifyColumnSQL[] = $newSQL;
```

Comment:
This was a hack. From the SQL syntax perspective, a column name followed by the column definition (which starts with the column name), this logic doesn't make sense. This code worked only because the column definition was generated for a column with empty name (which is also invalid).